certified-connectors/Data247logic/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3474 @@
+        "operationId": "Carrier247-Type",
+        "parameters": [
+          {
+            "name": "key",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Data247.com API Key",
+            "x-ms-summary": "Data247.com API Key"
+          },
+          {
+            "name": "phone",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Phone Number",
+            "x-ms-summary": "Phone Number"
+          }
+        ]
+      }
+    },
+    "/v3.0/CI/": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "response": {
+                  "type": "object",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "description": "status"
+                    },
+                    "results": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "phone": {
+                            "type": "string",
+                            "description": "phone"
+                          },
+                          "wless": {
+                            "type": "string",
+                            "description": "wless"
+                          },
+                          "carrier_name": {
+                            "type": "string",
+                            "description": "carrier_name"
+                          },
+                          "carrier_id": {
+                            "type": "string",
+                            "description": "carrier_id"
+                          },
+                          "country": {
+                            "type": "string",
+                            "description": "country"
+                          },
+                          "isoCountry2": {
+                            "type": "string",
+                            "description": "isoCountry2"
+                          }
+                        }
+                      },
+                      "description": "results"
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Carrier247 (International)",
+        "description": "Carrier247 (International)returns carrier information for international phone numbers, including whether they are wireless or landline numbers.",
```

Comment:
@Data247logic Please give spaces between "(International) and returns" in the value. Please fix all the typos in the all the files, if any.